msgraph/synchronization.go

```diff
@@ -0,0 +1,318 @@
+package msgraph
+
+import (
+	"context"
+	"encoding/json"
+	// "errors"
+	"fmt"
+	"io"
+	"net/http"
+)
+
+// SynchronizationJobClient performs operations on SynchronizationJobs.
+type SynchronizationJobClient struct {
+	BaseClient Client
+}
+
+// NewSynchronizationJobClient returns a new SynchronizationJobClient
+func NewSynchronizationJobClient(tenantId string) *SynchronizationJobClient {
+	return &SynchronizationJobClient{
+		BaseClient: NewClient(VersionBeta, tenantId),
```

Comment:
It looks like synchronization jobs are supported on the 1.0 API - if these work sufficiently can we change this default to target v1.0 instead?

Reply:
Unfortunately it doesn't work for version 1.0, I have tested some of the calls in the graph explorer for version 1.0 and they all gave me errors. Currently the documentation also is only available for the beta API.